### ISSUE TITLE
Not indenting array is not an error

### ISSUE THREAD

**Issue by rwxrob** [adrienverge/yamllint#374]
The following is being incorrectly flagged as an error:

```yaml
AnArray:
- Title: This is Fine
  Comment: Stop reporting this as an error
- Title: Considerations
  Comment: People should RTFS before making a "linter"
```

That is all.

**Comment by adrienverge** [comment]
Hello Rob Muhlestein,

Your problem is not very clear but it looks like you're not reporting a bug, rather expressing your personal taste on indentation. Some people like extra spaces at the end of lines, some others like dict keys to be unsorted, others don't use `---` markers. Still, there are yamllint rules for these use-cases: `trailing_spaces`, `key-ordering` and `document-start`. Your call to use them or not. Your call to configure them to do what you like.

Maybe you'll find the documentation interesting: especially [the `indent-sequences` part](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.indentation).

> People should RTFS before making a "linter"

OK

**Comment by jagibson** [comment]
@adrienverge while the OP was kind of rude, the YAML [failsafe schema](https://yaml.org/spec/1.2-old/spec.html#id2802662) does allow list items to be non-indented and recommends to do so "for generic YAML tools" - see [https://github.com/yaml/pyyaml/issues/234#issuecomment-591801490](https://github.com/yaml/pyyaml/issues/234#issuecomment-591801490) for discussion.  (In this specific case pyyaml is following the failsafe schema but yamllint is reporting it as invalid).

**Comment by perlpunk** [comment]
@jagibson this has nothing to do with the Failsafe schema or any YAML schema.
This is about the YAML syntax, and it's independent of the used schema.
Zero-indented lists are always valid.

The schema is about loading things like `true`, `123`, `null` etc. as certain types or as simple strings. In the failsafe schema everything is loaded as a string.
http://blogs.perl.org/users/tinita/2018/01/introduction-to-yaml-schemas-and-tags.html

PyYAML does not use the Failsafe schema by default, btw, because it implements YAML 1.1, where the word "schema" wasn't even used in the spec.
